views/fe-utils/fetchUtil.js

```diff
@@ -132,16 +138,44 @@ const deleteChart = async (id) => {
   return res.json()
 }
 
-export { 
+const editChartForm = async (formValues) => {
+  const res = await window.fetch(
+    `${routes.basePath}/api/v1/charts/${formValues._id}`,
+    {
+      ...defaultApiOptions,
+      method: 'PUT',
+      body: JSON.stringify(formValues),
+    }
+  )
+
+  if (res.status !== 200) return new Error(res.message)
+
+  return res.json()
+}
+
+const getChartForm = async (id) => {
```

Comment:
We're not getting/editing chart forms, we're getting/editing charts and rendering the data in a form. Slight difference but I think it's important.

```suggestion
const getChart = async (id) => {
```